layouts/index.redirects

```diff
@@ -0,0 +1,10 @@
+{{ $latest := site.Params.versions.latest }}
+/docs/latest     /docs/{{ $latest }}/
+/docs/latest/*     /docs/{{ $latest }}/:splat
+
+/docs/current     /docs/next/
+/docs/current/*     /docs/next/:splat
+
```

Comment:
We should eliminate current. That's an artifact of the way etcd.io was setup over time (IMHO), not really something we need to keep / setup here.

Reply:
I think you're right here. I'll remove it for this site.